DOCS-6273-update-docs (#156)
mulesoft/docs-api-manager: modules/ROOT/pages/org-credentials-config-mule4.adoc

include::_attributes.adoc[]
endif::[]

> Organization credentials provide a way to uniquely identify an organization, business group, or specific environment, when linking the Mule runtime to an Anypoint organization. Credentials consists of two keys: Client ID and Client Secret.Mule runtime uses these credentials to connect and access your Anypoint organization.

Reviewer: ```suggestion
Organization credentials provide a way to uniquely identify a specific environment, an organization, or a business group when linking Mule runtime engine (Mule) to an organization using Anypoint Platform. Credentials for environments, organizations, and business groups consist of two keys: the client ID and a client secret. Mule uses these credentials to connect to and access your organization.
```
Please review carefully. I suggested changes based on the following text about recommended connection. It seems like environment should be listed first, and business group last. However, we keep using just "organization," which makes it unclear that each of the three has its own ID and secret. Or did I misunderstand this?

Reviewer: Do not refer to "Anypoint organization." An organization uses Anypoint Platform, comprising various Anypoint-named tools, but there is no such thing as an Anypoint organization.

Do not capitalize client ID or client secret unless you are quoting a UI label, and then only in that specific context.

Reviewer: GLOBALLY search for and replace "Mule runtime" with "Mule runtime engine" or "Mule."
If you introduce it at "Mule runtime engine (Mule)," you can then use just "Mule."
"Mule runtime" is incorrect.

Reviewer: Also, should this intro text mention child organizations?

> The recommended way to link a Mule runtime to API Manager, is by using the environment Client ID and Secret. You may also use the organization Client ID and Secret, or even business group credentials if you want to track APIs defined in Business groups while the Mule runtime is linked to a parent Business group.

Reviewer: ```suggestion
The recommended way to link Mule to Anypoint API Manager (API Manager) is by using the environment client ID and secret. You may also use the organization client ID and secret, or even business group credentials if you want to track APIs defined in business groups linked to Mule or belonging to a parent business group linked to Mule.
```

> When you use environment credentials, you will be able to manage APIs that are specific to only that environment. If your APIs are in different environments of Anypoint Platform, you must specify the organization credentials of the organization that contains both environments.

Reviewer: ```suggestion
When you use environment credentials, you can manage APIs that are specific to only that environment. If your APIs are in different environments of Anypoint Platform, you must specify instead the credentials of the organization that contains both environments.
```

> You can also use parent organization credentials to manage child organization APIs. For example, if ACME is the master organization and Logistics is a business group that lives within ACME, your API Administrator will be able to track APIs created on Logistics business group using the credentials of both Logistics and of ACME. The is because ACME is the parent organization and Logistics is the child organization.

Reviewer: So, I am unclear on the hierachy here. Does an organization contain environments and business groups? Or is the order different? And should we mention child organizations as part of the hierarchy?
The text states that the recommended connection is with environment credentials...but then it goes on to say that that is limited by APIs being in different environments. Is it also limited by there being child organizations?

Reviewer: ```suggestion
You can also use parent organization credentials to manage child organization APIs. For example, if ACME is the parent organization and Logistics is a business group (child organization) within ACME, your API Administrator can track APIs created in Logistics using the credentials of either Logistics or of ACME. 
```
So...the child org. inherits the credentials of the parent org? Then why do business groups have separate client IDs and secrets?

> The identification of a unique resource with two keys, Client ID and Client Secret, is also used when identifying a client application. Client application credentials are not to be used as organization credentials. No harm is done if you choose to do so, but it will not link the Mule runtime to the organization.

Reviewer: ```suggestion
Although client applications also use client ID and client secret credentials, those are not used as organization credentials. No harm is done if you do so, but those credentials do not link Mule runtime to your organization.
```
This is confusing. Why don't we simply say that one should not use client app. credential as org. credentials? Seems to me if the same credentials are used for both app. and org., they would still work for connecting the org. to Mule....

> To pair your Mule runtime to your Anypoint Organization, the Runtime must start with Anypoint Platform credentials already configured. If you provision the adapter with organization credentials, you will be able to manage APIS on the specific adapter of that organization. 

Reviewer: ```suggestion
You cannot link your Mule runtime engine to your organization unless Mule is started using Anypoint Platform credentials already configured. If you provision an adapter with organization credentials, you can manage APIs on only that specific adapter of your organization. 
```

Reviewer: Please verify that the suggested rewording does not change your intended meaning.
"Pairing" = linking, right? Isn't this something that belongs earlier in this text?
How did an adapter come into the mix of things with credentials? Should it be mentioned along with org., environment, business group, and app., as "things that have their own client ID and secret"?

Author: kept pair

> == Obtaining Credentials

Reviewer: This container topic needs a text introduction that addresses the audience as "you" and reiterates why credentials are important. It should also prepare the audience for the subtopics within by mentioning them.

> === Obtaining Environment's Credentials (Recommended)

Reviewer: ```suggestion
=== Obtaining Environment Credentials
```
Follow with a text transition that reiterates that this is the recommended approach, and why.

. Log in to Anypoint Platform as an administrator, and click *Access Management*.

> . Click the *Environments* tab and click on the name of your environment within your desired organization.

Reviewer: ```suggestion
. Click the *Environments* tab and click the name of your environment within your desired organization.
```
Do not use "click on".
